unittests/test_progress_bar.py

```diff
@@ -28,19 +31,25 @@ def test_with_operation(pbclass):
 
 @pytest.mark.parametrize("pbclass", [ProgressBar, DummyProgressBar])
 def test_check_length_full(pbclass):
+    logger_utils.reset()
     p = pbclass(2, None)
-    with pytest.raises(Exception):
```

Comment:
This is (was) how you assert that an exception is thrown.

Reply:
reflected